src/errorOut.ts

```diff
@@ -0,0 +1,24 @@
+import logError = require("./logError");
+import { FirebaseError } from "./error";
+
+/**
+ * Errors out by calling `process.exit` with an exit code of 2.
+ * @param error an Error to be logged.
+ */
+export default function(error: Error): void {
+  let fbError: FirebaseError;
+  if (!(error instanceof FirebaseError)) {
```

Comment:
nit: invert this condition to make it clearer?

Reply:
okay, just for you :)